### PR TITLE
Add "Central" from BKP 2014

### PR DESCRIPTION
**Pull request by frozencemetery**
This is my problem for which I am adding source, and linking to my own blog post.  Thanks!

### CODE REVIEW

File: boston-key-party-2015/pwning/central-square/54.152.100.187/index.html [hunk 1]
```diff
@@ -1,10 +0,0 @@
-
-<html><head>
-  <title>Acme scrambling service</title>
-</head><body>
-  <form method="post" name="form">
-    Where? <input name="location" type="text">
-   <input name="Submit" type="submit" value="Submit">
-  </form>
-</body></html>
-
```
Comment (dhanvi) [inline boston-key-party-2015/pwning/central-square/54.152.100.187/index.html]: I am not sure why you removed this file :-1:

**Comment by frozencemetery** [comment]
@dhanvi It's because it isn't a file in my problem.  There is no index.html; the entire website is served dynamically from server.py.  Please remove this file.

**Comment by frozencemetery** [comment]
@dhanvi Take a look - it's right here: https://github.com/ctfs/write-ups-2015/blob/master/boston-key-party-2015/pwning/central-square/54.152.100.187/server.py#L132-139

**Comment by dhanvi** [comment]
sure @frozencemetery, sorry for that :disappointed:

**Comment by frozencemetery** [comment]
Thanks!

**Pull request merged**